code/style.css

```diff
@@ -1,7 +1,326 @@
-/* the + sign is used to select the sibling that comes immediately next to the element you selected.
-in this case it is the div with the answer class next to the div with the active class.
-this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
+* {
+  box-sizing: border-box; 
+  }
+
+body {
+  font-family: Arial, Helvetica, sans-serif;
+  margin: 0 auto;
+  padding: 0 auto;
+  width: 100%;
+  color: #444;
+  background-color: #eee;
+} 
+
+h2 {
+  font-size: 20px;
+}
+
+h3 {
+  font-size: 32px;
+}
+
+/***Styling header/hero***/
+.hero {
+  position: relative;
+}
+
+.hero-menu {
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  right: 0;
+  left: 0;
+  display: flex;
+  justify-content:flex-start;
+  align-items: flex-start;
+}
+
+.topnav {
+  overflow: hidden;
+  background-color:none;
+  margin: 50px 0px 0px 20px;
+}
+
+.topnav a {
+  color: #eee;
+  text-align: center;
+  padding: 14px 16px;
+  text-decoration: none;
+  font-size: 20px;
+}
+
+.topnav a:hover {
+  background-color: rgb(158, 207, 207);
+  color: black;
+}
+
+.hero-title {
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  right: 0;
+  left: 0;
+  color: #eee;
+  font-size: 85px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.hero-image {
+  width: 100vw;
+  height: 100vh;
+  object-fit: cover;
+  top:0;
+  left: 0;
+}
+/***Intro section***/
+
+.introduction {
+  padding: 15px 100px;
+  text-align: center;
+}
+
+.introduction h2 {
+  font-size: 15x;
+}
+
+/***F&Q and Form section***/
+
+.information {
+   display: flex;
+   flex-direction: row;
+   justify-content: center;
+   flex-wrap: wrap;
+}
+
+.information h2 {
+  font-size: px;
+}
+
+.container {
+   width: 45%;
+    }
+
+/***Accordion section***/
+
+.accordion {
+   display: flex;
+   flex-direction: column;
+   justify-content: center;
+   padding: 20px 50px 120px;
+   margin: 20px;
+   background-color: rgb(106, 165, 165);
```

Comment:
is there a reason for using rgb in some places and hexcolors in some? 
Try to stick to one approach if possible.